templates/pro/index.html

```diff
@@ -42,22 +42,22 @@ <h2>Same great OS.<br />More security updates.</h2>
       <div class="row">
         <h2 class="p-text--x-small-capitalised">Security coverage compared</h2>
       </div>
-      <div class="row row-card">
-        <div class="col-3 col-medium-3 col-card bg-light">
+      <div class="row p-tiered-cards">
+        <div class="col-3 col-medium-3 p-tiered-card--low">
           <p class="p-heading--5">LTS</p>
           <p class="p-heading--1">
             Main:<br />5 years
           </p>
           <p>2,300 packages in the Ubuntu Main repo supported for 5 years</p>
         </div>
-        <div class="col-3 col-medium-3 col-card bg-mid-dark">
+        <div class="col-3 col-medium-3 p-tiered-card--medium">
           <p class="p-heading--5">Pro (Infra-Only)</p>
           <p class="p-heading--1">
             Main:<br />10 years
           </p>
           <p>2,300 packages in the Ubuntu Main repo supported for 10 years</p>
         </div>
-        <div class="col-6 col-medium-6 col-card bg-dark">
+        <div class="col-6 col-medium-6 p-tiered-card--high">
```

Comment:
I appreciate the effort to make this responsive, but putting the third card separately no longer works as a comparison visually - I would use col-medium-2 on all - or make therm all stretch full width and create layuot internally by using 2 columns for the heading, 4 for the text for example.

Reply:
Done